src/_includes/docs/install/flutter/download.md

```diff
@@ -84,6 +90,9 @@
 
 [SDK archive]: {{site.url}}/release/archive
 
-{% if os == 'Windows' %}
-{% include docs/install/reqs/windows/set-path.md terminal=terminal %}
-{% endif %}
+{% case os %}
+{% when 'Windows' %}
+{% include docs/install/reqs/windows/set-path.md terminal=terminal target=target %}
+{% when 'macOS' %}
+{% include docs/install/reqs/macos/set-path.md terminal=terminal target=target %}
+{% endcase %}
```

Comment:
I don't know if it's just me but this structure is hard to review with all of these includes. I've noticed that the size of the `_includes` directory is increasing and I'm not sure that's a great trend for maintainability overall. If there are places where we can just add the docs in-line I think that would be better in the long term. 

Includes are great for reusable snippets of instructions but might have diminishing returns if we use them too much.

Reply:
It can be on first pass, but future updates may be easier. Most of these types of pages get a lot of reuse as most (~75%) of the content is the same for all 4 development platforms.

Reply:
It's _really_ hard for me to follow what's going on. I have to jump around from file to file rather than just read it in-line.